packages/deployer/README.md

```diff
@@ -1,11 +1,18 @@
 # @api3/deployer
 
-> Airnode deployment CLI
+> Airnode deployment CLI and its docker image
 
-**The recommended way to deploy Airnode is by using the [Deployer Docker](../../docker/README.md) image. This package
-simply implements the CLI used by that image and is not meant to be used directly by the end user.**
+## User documentation
```

Comment:
I thought a bit about how the READMEs should look like and I quite like the structure:
```
# package name

> description

## User documentation

link to api3 docs

## For developers

...
```